db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.1].define(version: 2025_03_10_152913) do
+ActiveRecord::Schema[7.2].define(version: 2025_03_10_152913) do
```

Comment:
Is this change required? Just curious.

Reply:
Sorry @jrmhaig , I only just saw your comment. That is a good question. It created it when I did the migration and I when I rolled back it remained and I wasn't sure whether to take it out.